lib/js/components/SurveyToggle/SurveyToggle.vue

```diff
@@ -36,14 +37,62 @@
 @import '../../../styles/settings/spacings';
 @import '../../../styles/settings/shadows';
 @import '../../../styles/settings/typography';
-
-$survey-toggle-size: 48px;
-
-$color-disabled-neutral: #000; // IT WILL BE REPLACED IN NEXT PR
+@import '../../../styles/settings/colors/tokens';
+@import '../../../styles/settings/survey-toggle';
+@import '../../../styles/components/survey-toggle';
 
 .surveyToggle {
 	$self: &;
 
+	@each $color-name, $color-map in $survey-toggle-colors {
+		&.-#{$color-name} {
+			#{$self}__toggle {
+				@include setSurveyToggleNormalState(
```

Comment:
IMO, it makes sense to seaprate mixins for `toggle` and `ring`. Parameter sets are separate for both use-cases, so there is no reason to complicate it

Reply:
Right. Refactored.